### PR TITLE
Add an example of logging with tflog.

### PR DESCRIPTION
**Pull request by paddycarver**
Add an example of writing log output using the tflog package.

### CODE REVIEW

**Review by bflad — APPROVED** [review]
Looks good to me, but will defer to @hashicorp/terraform-education if the Learn tutorial should also be updated as part of adding this.

File: hashicups/resource_order.go [hunk 1]
```diff
@@ -124,6 +125,10 @@ func (r resourceOrder) Create(ctx context.Context, req tfsdk.CreateResourceReque
 		return
 	}
 
+	// for more information on logging from providers, see
+	// https://pkg.go.dev/github.com/hashicorp/terraform-plugin-log/tflog
```
Comment (laurapacilio) [inline hashicups/resource_order.go:129]:
```suggestion
	// for more information on logging from providers, refer to
	// https://pkg.go.dev/github.com/hashicorp/terraform-plugin-log/tflog
```

**Review by laurapacilio — COMMENTED** [review]
Just requesting that we update the language to not use "see", per our style guide :)

**Comment by im2nguyen** [comment]
lgtm, @tr0njavolta, do you want to merge?

**Review by im2nguyen — APPROVED** [review]

**Review by tr0njavolta — APPROVED** [review]

**Pull request merged**